pacman/operations/partition_algorithms/partition_and_place_partitioner.py

```diff
@@ -128,7 +129,8 @@ def _partition_vertex(
                 PartitionerMaximumSizeConstraint)
             for constraint in max_atom_constraints:
                 possible_max_atoms.append(constraint.size)
-        max_atoms_per_core = min(possible_max_atoms)
+
+        max_atoms_per_core = int(min(possible_max_atoms))
 
```

Comment:
yea, pycharm seems to think min() returns a iterable. probably python doc issue. pycharm uses it to tell me off when we put stuff with wrong types into calls. is why you see :type param: blah or None a lot